functions/billing/index.js

```diff
@@ -19,7 +19,11 @@
 const {google} = require('googleapis');
 const {auth} = require('google-auth-library');
 
-const PROJECT_ID = process.env.PROJECT_ID;
+let PROJECT_ID = process.env.PROJECT_ID;
+// Attempt to use GCP_PROJECT if no PROJECT_ID was specified
+if (typeof PROJECT_ID === 'undefined') {
+  PROJECT_ID = process.env.GCP_PROJECT;
+}
```

Comment:
More idiomatic:

```js
const PROJECT_ID = process.env.PROJECT_ID || process.env.GCP_PROJECT;
```

However, within GCF, we can rely on just the `GCP_PROJECT` if I'm not mistaken: https://cloud.google.com/functions/docs/env-var

Reply:
I was thinking of making the logic more explicitly readable but I can change if that's preferred.

Can't depend on GCP_PROJECT for Node 10: https://cloud.google.com/functions/docs/env-var#nodejs_10_and_subsequent_runtimes

Reply:
Or maybe it makes more sense to just tell them to set GCP_PROJECT manually? I was under the impression that would throw an error but it only seems to for older runtimes.

Reply:
I would just use the code snippet I suggested.
It's the same logic and should be safe in case we only use one or the other env vars.

I don't know where this snippet is added in the docs, but we could check there too to see what other languages do.

Reply:
Understood, I'll simplify the code a bit and explain in the docs: https://cloud.google.com/billing/docs/how-to/notify

Reply:
We've dropped support for Node versions < 10 in our docs.

I'd drop the `GCP_PROJECT` here.